src/toffy/normalize.py

```diff
@@ -707,9 +728,11 @@ def normalize_fov(img_data, norm_vals, norm_dir, fov, channels, extreme_vals):
             "recommended: {}".format(fov, bad_channels)
         )
 
-    # correct images and save
+    # correct images and save, ensure that no division by zero happens
     norm_vals = norm_vals.astype(img_data.dtype)
-    normalized_images = img_data / norm_vals.reshape((1, 1, 1, len(norm_vals)))
+    norm_div = norm_vals.reshape((1, 1, 1, len(norm_vals)))
+    norm_vals_masked = np.ma.masked_values(x=norm_vals, value=0)
+    normalized_images = np.ma.divide(img_data.values, norm_div).filled(0)
```

Comment:
Is this divide by zero stuff happening as a result of skipping the masses with 0 MPH, or it was already a problem but it's only now becoming apparent?

Reply:
This is happening because we're skipping masses with 0 MPH.